acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/LoginPage.java

```diff
@@ -0,0 +1,63 @@
+package uk.nhs.digital.ps.test.acceptance.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import uk.nhs.digital.ps.test.acceptance.webdriver.WebDriverProvider;
+
+public class LoginPage extends AbstractPage {
+
+
+    public LoginPage(final WebDriverProvider webDriverProvider) {
+        super(webDriverProvider);
+    }
+
+    public void open() {
+        getWebDriver().get(URL);
+    }
+
+    public void loginWith(final String username, final String password) {
+        getUsernameField().sendKeys(username);
+        getPasswordField().sendKeys(password);
+        getLoginButton().click();
+    }
+
+    private WebElement getLoginButton() {
```

Comment:
It's a small thing, but calling this "find", rather than "get", would more clearly indicate the behaviour to me. i.e. it's not just returning a value that's already stored, but instead going out and hunting it down. Same comment for other equivalent "get" methods.

Reply:
Good point. I'll rename the methods.